physics/mp_thompson.F90

```diff
@@ -594,10 +628,10 @@ subroutine mp_thompson_run(ncol, nlev, con_g, con_rd,        &
                               diagflag=diagflag, do_radar_ref=do_radar_ref_mp,               &
                               re_cloud=re_cloud, re_ice=re_ice, re_snow=re_snow,             &
                               has_reqc=has_reqc, has_reqi=has_reqi, has_reqs=has_reqs,       &
-                              rand_perturb_on=rand_perturb_on, kme_stoch=kme_stoch,          &
+                              rand_perturb_on=spp_mp, kme_stoch=kme_stoch,                   &
```

Comment:
Weiwei - do you know if this is a namelist parameter/setting (the spp_mp flag)?  Or, how it is set?

Reply:
Thanks Laurie for reviewing them! Yes, I think this is a namelist parameter to let people choose which variable they want to add the perturbation. Please feel free to check Line 1188-1212 in module_mp_thompson.F90. I've added this to the GFS_layer and submitted a PR for your review.

Reply:
spp_mp has been set as 7 in this code and passed to mp_thompson_run.